mynah-ui/src/mynah-ui/ui/messages/handler.ts

```diff
@@ -31,13 +31,6 @@ export class TextMessageHandler {
         this.mynahUI.addChatItem(tabID, {
             type: ChatItemType.PROMPT,
             body: chatPrompt.escapedPrompt,
-            ...(chatPrompt.attachment !== undefined
```

Comment:
why are these not needed anymore?

Reply:
Those were the link attachments, since they're removed from the library it should be removed from the usages too. This is already a major version bump for mynah-ui

Reply:
Do we know the blast radius on this major version bump? Does this change affect others teams integration with Q chat?